### PR TITLE
Add BThome integration

### PR DESCRIPTION
**Pull request by Ernst79**
## Proposed change

Add documentation for new BThome integration. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [#77224](https://github.com/home-assistant/core/pull/77224)
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3611
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by balloob — DISMISSED** [review]

File: source/_integrations/bthome.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- Energy
+- Power
+- PM2.5
+- PM10
+- CO2
+- VOC
+
+## Encryption
+
+BThome optionally supports AES encryption to protect the sensor data that are broadcasted. You will need to set a 32 character hexadecimal (16 bytes) key to decrypt the data, when using encryption. This key is called the bindkey.
```
Comment (MartinHjelmare) [inline source/_integrations/bthome.markdown]:
```suggestion
BThome optionally supports AES encryption to protect the sensor data that is broadcasted. You will need to set a 32 character hexadecimal (16 bytes) key to decrypt the data, when using encryption. This key is called the bindkey.
```

File: source/_integrations/bthome.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## ATC pvvx firmware support
```
Comment (MartinHjelmare) [inline source/_integrations/bthome.markdown]: I'd keep the firmware details on the bthome site and not duplicate it here.

File: source/_integrations/bthome.markdown (hunk 1 above)
Comment (Ernst79) [reply source/_integrations/bthome.markdown]: fixed

File: source/_integrations/bthome.markdown (hunk 2 above)
Comment (Ernst79) [reply source/_integrations/bthome.markdown]: Removed

**Review by klaasnicolaas — DISMISSED** [review]
✅ LGTM

File: source/_integrations/bthome.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+{% include integrations/config_flow.md %}
+
+The BThome BLE integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
+
+BTHome is an energy effective but flexible BLE format to broadcast data with Bluetooth and allows you to create your own DIY BLE sensors. More information about the BThome BLE format and projects that use the format can be found on the [BThome website](https://bthome.io/).
+
+## Supported sensor measurements
+
+The current release supports the following sensor measurement types. More (binary) sensor types will be added in a future release.
```
Comment (balloob) [inline source/_integrations/bthome.markdown]: Let's remove this here. If we want, we can add a line below this list where people can request new sensors.

```suggestion
The current release supports the following sensor measurement types.
```

I actually don't even know if we should mention this all here, it's all in the BTHome spec?

File: source/_integrations/bthome.markdown [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- Pressure
+- Illuminance
+- Energy
+- Power
+- PM2.5
+- PM10
+- CO2
+- VOC
+
+## Encryption
```
Comment (balloob) [inline source/_integrations/bthome.markdown]: Same here, shouldn't we just point people to the spec?

File: source/_integrations/bthome.markdown (hunk 3 above)
Comment (Ernst79) [reply source/_integrations/bthome.markdown]: yes, but the BThome specs are also giving binary sensors, which are not supported in the bthome integration yet, but they are in e.g. BLE monitor. I hope to finish the other sensors before beta release, but binary sensors will follow probably in October release.

File: source/_integrations/bthome.markdown (hunk 4 above)
Comment (Ernst79) [reply source/_integrations/bthome.markdown]: I think we should at least tell people they will be prompted to enter a key, shouldn't we? I rephrased it a bit, and made reference to the specs.

File: source/_integrations/bthome.markdown (hunk 3 above)
Comment (Ernst79) [reply source/_integrations/bthome.markdown]: If you leave the PR open for a few more days, I can try to add the other sensors, such that we can say that we only support sensors in the current release. We can than leave the list out. But I need two or three days to add the remaining sensors.

File: source/_integrations/bthome.markdown [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+- Energy
+- Power
+- PM2.5
+- PM10
+- CO2
+- VOC
+
+## Bindkey
+
+When using encryption for your BThome sensor, you will promted to enter your 32 character hexadecimal (16 bytes) encryption key. This key is called the bindkey. More information about the bindkey can be found in the [specifications](https://bthome.io/).
```
Comment (balloob) [inline source/_integrations/bthome.markdown]:
```suggestion
When using encryption for your BThome sensor, you will promted to enter your 32 character hexadecimal (16 bytes) encryption key. This key is called the bindkey. More information about the bindkey can be found in the [specifications](https://bthome.io/#encryption).
```

**Review by balloob — APPROVED** [review]

**Pull request merged**